src/database/migrations/20220321123904-create-trip.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable valid-jsdoc */
+/* eslint-disable require-jsdoc */
+export async function up(queryInterface, Sequelize) {
+    await queryInterface.createTable('trips', {
+      id: {
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
+        allowNull: false,
+        primaryKey: true
+      },
+      departure: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      destination: {
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      dateOfTravel: {
+        type: Sequelize.DATE,
+        allowNull: false
+      },
+      dateOfReturn: {
+        type: Sequelize.DATE,
+        allowNull: true
+      },
+      travelReason: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      status: {
+        type: Sequelize.STRING,
```

Comment:
Change this to match to the type in model

Reply:
Yap please do this .Make status enum as well with the migration. Otherwise good job